form-samples/two-column-logo-header/README.md

```diff
@@ -0,0 +1,39 @@
+# Dashed Line Header
```

Comment:
Please change the title to this sample title.

Reply:
I've added Sample Title to the json file and the images.   Hopefully I've understood you're request.   let me know.